tests/suites/test_suite_asn1write.data

```diff
@@ -90,3 +90,69 @@ mbedtls_asn1_write_len:16909060:"8401020304":5:5
 
 ASN.1 Write / Read Length #12 (Len = 16909060, buffer too small)
 mbedtls_asn1_write_len:16909060:"8401020304":4:MBEDTLS_ERR_ASN1_BUF_TOO_SMALL
+
```

Comment:
All the tests below have the leading bits all set, and the following unused bits unset. Can we mix it up a bit, and have some leading bits unset, and some unused bits set?

Reply:
Right, sorry about that, I forgot to add tests for the other function.